src/types/index.ts

```diff
@@ -533,6 +535,16 @@ export type ApiDisbursement = {
   status_history: ApiDisbursementHistory[];
   created_at: string;
   updated_at: string;
+  created_by: {
+    id: string;
+    first_name: string;
+    last_name: string;
+  };
+  started_by: {
+    id: string;
+    first_name: string;
+    last_name: string;
+  };
```

Comment:
If `createdBy` and `startedBy` can be optional (lines 213 and 214 above), does it mean these could also be optional?